web/src/pages/Metrics/locales/en-US.ts

```diff
@@ -17,5 +17,5 @@
 export default {
   'page.metrics.empty.description.grafanaNotConfig': 'You have not configured Grafana',
   'page.metrics.button.grafanaConfig': 'Configure',
-  'page.metrics.tip': 'Use browser iframe to store monitor page URL, local only.',
+  'page.metrics.tip': 'Use browser localStorage to store monitor page URL, local only.',
 };
```

Comment:
Should I change the `'page.metrics.xxx` to `page.dashboard.xxx` and rename the folder?

Reply:
would better to use `metrics` all the paces